package.json

```diff
@@ -2,17 +2,21 @@
   "name": "react-dropzone",
   "description": "Simple HTML5 drag-drop zone with React.js",
   "main": "dist/index.js",
+  "module": "dist/es/index.js",
   "scripts": {
     "ci": "git-cz",
     "clean": "rimraf ./dist",
-    "build": "npm run clean && webpack",
+    "build": "npm run clean && webpack && npm run build:es",
```

Comment:
I'd prefer we have 2 separate `build` scripts: `build:umd` and `build:es` to make the intent clear.

`build:umd` would be `NODE_ENV=production webpack`

Reply:
I'll do this and get back to you